force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,27 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
+        String str = 'Hello, Salesforce!';
+
         // Get the length of the String        
+        Integer strLength = str.length();
+
         // Get a substring of the String of the first 5 characters
+        String subStr = str.substring(0, 5);
+
         // Convert the String to lower case
+        String lowerStr = str.toLowerCase();
         
         // Return the list with the string data
         // In the first position add the string length
         // In the second position add the substring
         // In the third position add the string in lower case
+        List<String> strList = new List<String>();
+        strList.add(strLength.toString());
+        strList.add(subStr);
+        strList.add(lowerStr);
 
-        return null; // Replace null with the variable you used to store the result
+        return strList; // Replace null with the variable you used to store the result
```

Comment:
great work. If you want you for a lot of list problems you can be more concise and skip creating variables you don't really need. 

```
 public static List<String> practiceStringMethods() {
        // Initialize a String with 'Hello, Salesforce!'

        // Get the length of the String
        // Get a substring of the String of the first 5 characters
        // Convert the String to lower case

        // Return the list with the string data
        // In the first position add the string length
        // In the second position add the substring
        // In the third position add the string in lower case
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
```